src/print.hpp

```diff
@@ -1,9 +1,13 @@
 #ifndef PRINT_HPP
 #define PRINT_HPP
 
+#ifdef _WIN32
+#define __PRETTY_FUNCTION__ __FUNCTION__
+#endif
+
 #include <stdio.h>
 #define print_error(...)																	\
-	fprintf(stderr, "\e[36m%s: \"%s\" %i:\e[0m ", __FILE__, __PRETTY_FUNCTION__, __LINE__);		\
```

Comment:
why is this thing removed, it is ascii escape sequences needed for painting colours

Reply:
\e is not portable and windows compile errors